src/ddns.c

```diff
@@ -149,7 +149,7 @@ static int server_transaction(ddns_t *ctx, ddns_info_t *provider)
 	trans->rsp         = ctx->work_buf;
 	trans->max_rsp_len = ctx->work_buflen - 1;	/* Save place for terminating \0 in string. */
 
-	logit(LOG_DEBUG, "Querying DDNS checkip server for my public IP#: %s", ctx->request_buf);
+	logit(LOG_DEBUG, "Querying DDNS checkip server for my public IP#:\n%s", ctx->request_buf);
 
```

Comment:
I understand this works for you, but for streaming over the Internet to a remote syslog server it does not, so NAK

Reply:
Every place that I've added the linefeed returns no less than 12-15 lines of multi-line output. What do we gain by having output that looks like this?

```
Jan 12 23:42:21  inadyn[69789] <Debug>: Response: HTTP/1.1 200 OK
	Date: Mon, 13 Jan 2020 07:42:21 GMT
	Content-Type: application/json
	Connection: close
	Set-Cookie: __cfduid=dde8b5d62d1d5be7005d801db88b5e4eb1578901341; expires=Wed, 12-Feb-20 07:42:21 GMT; path=/; domain=.api.cloudflare.com; HttpOnly; SameSite=Lax
	CF-Ray: 5545bba7dcc9933a-SJC
	Cache-Control: no-store, no-cache, must-revalidate, post-check=0, pre-check=0
	Expires: Sun, 25 Jan 1981 05:00:00 GMT
	Strict-Transport-Security: max-age=31536000, max-age=31536000
	CF-Cache-Status: DYNAMIC
	Expect-CT: max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"
	Pragma: no-cache
	Served-In-Seconds: 0.030
	X-Content-Type-Options: nosniff
	X-Frame-Options: SAMEORIGIN
	Server: cloudflare

	{"result":[{"id":"aad8dfa1218ae29dbdfe5fa9569cb682"
```

Instead of like this?

```
Jan 12 23:42:21  inadyn[69789] <Debug>: Response:
	HTTP/1.1 200 OK
	Date: Mon, 13 Jan 2020 07:42:21 GMT
	Content-Type: application/json
```

We're talking about one linefeed character addition to a long multiline string.

Reply:
It's a matter of both personal preference and the fact that \n usually does not survive sending to a remote syslog server. In case you want the second output I recommend going with two calls to logit() instead, as below.

Reply:
that doesn't produce the desire effect. It would then appear like so:

```
Jan 12 23:42:21  inadyn[69789] <Debug>: Response:
Jan 12 23:42:21  inadyn[69789] <Debug>: HTTP/1.1 200 OK
	Date: Mon, 13 Jan 2020 07:42:21 GMT
	Content-Type: application/json
	Connection: close
```

...which is wasteful and uninformative

Reply:
I understand your point of view.  Ideally "Response: HTTP code" and the next line "Details: ... ", but that's too much work for such a small thing.  I'd rather keep it as-is.